tests/lib/nested.sh

```diff
@@ -90,11 +90,13 @@ create_nested_core_vm(){
 }
 
 start_nested_core_vm(){
+    local QEMU=$(get_qemu_for_nested_vm)
     systemd_create_and_start_unit nested-vm "${QEMU} -m 2048 -nographic \
         -net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22 \
         -drive file=$WORK_DIR/ubuntu-core.img,if=virtio,cache=none,format=raw \
         -drive file=${PWD}/assertions.disk,if=virtio,cache=none,format=raw \
-        -monitor tcp:127.0.0.1:$MON_PORT,server,nowait -usb"
+        -monitor tcp:127.0.0.1:$MON_PORT,server,nowait -usb \
+        -machine accel=kvm"
```

Comment:
Yay!

Reply:
BTW. I think this can be `-accel kvm` but I guess the current code woks too.